k-distribution/k-tutorial/1_basic/19_debugging/README.md

```diff
@@ -69,6 +69,8 @@ command listed in this output message and add the corresponding
 Please note that the path will be different on your machine than the one
 listed above. 
 
+> *Note* The `krun --debugger` command will launch the `lldb` debugger on macOS instead of `gdb`. See https://lldb.llvm.org/use/map.html for a mapping between `gdb` and `lldb` commands. 
```

Comment:
I'd mention it here

Reply:
do you want to add the lldb support and leave this open util then?

Reply:
Yeah I think it's very much worth doing, but I don't know how straightforward the port will be. I'll open an issue

Reply:
https://github.com/runtimeverification/k/issues/2650